decidim-comments/app/packs/stylesheets/comments.scss

```
@@ -28,11 +28,35 @@
   .author__avatar-container {
     @apply w-8 h-8;
   }
+
+  .comments-section {
+    &__header {
+      @apply flex items-center justify-center space-x-2 p-4 bg-secondary/5 rounded-b-md mb-4;
+
+      svg {
+        @apply w-6 h-6 mr-2 fill-secondary;
+      }
+    }
+
+    &__title {
+      @apply text-md font-semibold text-secondary;
+    }
+  }
+}
+
+.comments-two-columns {
+  .most-upvoted-label {
+    @apply px-2 mb-2;
+  }
 }
 
 .comment {
   @apply rounded-lg border-2 border-background-5 p-3 pl-6;
 
+  &.top-comment {
+    border-color: #3e4c5c;
```

I do not like this. We should rely on what tailwind gives us. 
```suggestion
    @apply border-gray-2;
```